tests/common/osutil/test_default.py

```diff
@@ -570,6 +577,14 @@ def test_is_current_instance_id_from_file(self, mock_read, mock_isfile):
         self.assertTrue(util.is_current_instance_id(
             "B9F3C233-9913-9F42-8EB3-BA656DF32502"))
 
+        mock_read.return_value = "b9f3c233-9913-9f42-8eb3-ba656df32502"
+        self.assertTrue(util.is_current_instance_id(
+            "B9F3C233-9913-9F42-8EB3-BA656DF32502"))
+
+        mock_read.return_value = "33c2f3b9-1399-429f-8eb3-ba656df32502"
+        self.assertTrue(util.is_current_instance_id(
```

Comment:
This would fail - It should be `assertFalse`

Reply:
Why is that?  It's just the lowercase variant of the test on new line 576?

Reply:
I assume you are trying to match Guids, but they don't match - `33c2f3b9-1399-429f-8eb3-ba656df32502` wont match `B9F3C233-9913-9F42-8EB3-BA656DF32502`. Please let me know if I am missing something.

Reply:
is_current_instance_id checks for an exact match or for an endian conversion match on the first three components of the GUID to deal with ambiguity in interpreting the specification... look at the other comparisons that exist in the file.  Did I enter the endian swapped data wrong?